WIP: prompt tweaks

jekabs-karklins opened the pull request:
This PR aims to create PR generated description less verbose and skips generating "How Has This Been Tested?" altogether if unsure

jekabs-karklins committed:
Update autofill_description.py

fgandellini reviewed (approved):
Looks good to me! Thanks for the changes. Let's try them out! :)

Commit:
chore(autofill_description.py): update comments and formatting in the main function
docs(autofill_description.py): update pull request description template to provide clearer instructions and sections
